lib/perl/Genome/InstrumentData/Solexa.pm

```diff
@@ -1253,10 +1253,9 @@ sub get_default_alignment_results {
 
     unless(@alignment_results) {
         # Get alignment results for this inst data and the default aligner name
-        my $pp = Genome::ProcessingProfile::ReferenceAlignment->default_profile;
         @alignment_results = Genome::InstrumentData::AlignmentResult->get(
             instrument_data_id => $self->id,
-            aligner_name       => $pp->read_aligner_name,
+            aligner_name       => 'bwa',
```

Comment:
I agree it's weird to have the hard-coded "bwa" in here.  It's intended to preserve the value from the old processing-profile `2635769`.

All the code in this method inside this `unless` block is an ugly heuristic for old data not in analysis projects, so I believe it should keep the old value (`bwa`) regardless of what the new default might be.

My preference would be to delete this `unless` block entirely!  LIMS developed this code and is its primary user, so it's really up to them, though.